### DISCUSSION TITLE
Block comments improvement

### DISCUSSION THREAD

**Discussion by TomPlanche** [zed-industries/zed#18451] [Feature Requests]
Hello everyone.

I'd love to have block comment autofill.

I often use Jetbrains IDEs and when I type `/**` and hit `return`, it autocompletes to
```
/**
 * |
 */
``` 

Where `|` is the cursor position.

I've looked around the code in `crates/languages/src/typescript/config.toml` and tried to change the 

```{ start = "/*", end = " */", close = true, newline = false, not_in = ["string", "comment"] },```

to 

```{ start = "/**", end = " */", close = true, newline = true, not_in = ["string", "comment"] },```

 but all I have is:

 ```
 /**| */
 ```

I'd also love to have automatic new line completion when hitting `return` in a block comment.

```
/**
 * Some text|<- return hit here
 * |<- cursor now here
 */
``` 

I suppose that the `config.toml`'s `bracket` fiels is treated by the `language.rs` file.

How can I work with this file (or an other) to make the comment block work as I want ?

Thanks !

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
